fgo_mat_counter.py

```diff
@@ -209,17 +229,21 @@ def crop_edges(targetImg):
         min_x, max_x = min(x, min_x), max(x + w, max_x)
         min_y, max_y = min(y, min_y), max(y + h, max_y)
     
+    # 0.04 chosen because of the black and blue border image. Too high and the image won't get cropped
+    # Too low and other images will get cropped unnessarily.
+    if (min_x/width < 0.04 or abs(width - max_x) < 0.04)/width and\
+        (min_y/height < 0.04 and abs(height - max_y)/height < 0.04):
+        logging.debug("Abort color crop")
+        return targetImg
+

```

Comment:
I added this to bring the top up when there was no border to crop at the top:
`min_y = min_y if height - min_y > 20 else 0`

Reply:
I can add `min_y = min(0, min_y)` before the force 16:9 bro.

Reply:
Unfortunately where I clicked the '+' is not indicated in the comment but I inserted that code above line 234.

When detecting the top of the game screen via findContours the edge closest to the top of the image can be up to 25 pixels, in the worst case, below the top of the game screen so the idea is to just use the top of image if the detected top (min_y) is within some range (20 pixels in in this case) of the top of image. This should cover the no border dark and light background scenario where the highest edge is the drop count or some portion of the background above it.
If there is a top border then min_y will be much further from the top of image and I at least don't have a method for detecting if the edge was the top of game screen (light background) or the drop count ~20 pixels lower (dark background).